lib/cocoapods/command/list.rb

```diff
@@ -26,11 +26,9 @@ def run
       end
 
       def update_if_necessary!
-        if @update && config.verbose?
-          UI.section("\nUpdating Spec Repositories\n".yellow) do
-            Repo.new(ARGV.new(['update'])).run
-          end
-        end
+        UI.section("\nUpdating Spec Repositories\n".yellow) do
+          Repo.new(CLAide::ARGV.new(['update'])).run
+        end if @update
```

Comment:
does  update action should depends on verbose option here ?

Reply:
it was before I don't think we should change this currently.

Reply:
👌,  I will revert the change . 

however,  i think it is confusing that `pod list --update` command does't update ,  but `pod list --update  --verbose` command does. How do you think?

Reply:
hey @dnkoutso  

`config.verbose`  is set in `Command initialize` method when verbose is not nil.
if we don't pass `config.verbose?` as a parameter to `Repo::Update.new(CLAide::ARGV.new([]))`, 
`Repo::Update` command will not set `config.verbose`, and it will get the same verbose value  as `List`.

```
def initialize(argv)
  super
  config.silent = argv.flag?('silent', config.silent)
  config.verbose = self.verbose? unless verbose.nil?
  unless self.ansi_output?
    Colored2.disable!
    String.send(:define_method, :colorize) { |string, _| string }
  end
end
```
so, shall I just remove `&& config.verbose?` ?

Reply:
yeah I think `Config` is a singleton anyway so it will be OK.

Please delete it and update which is basically what you had :) thanks again.